voting/urls.py

```diff
@@ -5,6 +5,8 @@
 urlpatterns = patterns('voting.views',
     url(r'^$', 'voting',
         name='voting_index'),
+    url(r'^(?P<vote_token>[0-9]+)/$', 'voting',
+        name='voting_index'),
 )
```

Comment:
Don't see the problem really. You can explicitly reverse them with different params, will work just fine.